File: Easy/1929 - concatenationofarray/concatenationofarray.js
var getConcatenation = function(nums) {
    const         size = nums.length;
    const         result = new Array(2 * size);

    let i = -1;
    while (++i < size)
    {
        result[i] = nums[i];
        result[i + size] = nums[i];
    } 
    return result;
};

File: Easy/1929 - concatenationofarray/concatenationofarray.cpp
class Solution {
    public:
        vector<int> getConcatenation(vector<int>& nums) {
            int         size = nums.size();
            vector<int> result(2 * size);

            for (int i = 0; i < size; i++)
            {
                result[i] = nums[i];
                result[i + size] = nums[i];
            } 
            return result;
        }
};